docs/content/preview/tutorials/azure/azure-app-service.md

```diff
@@ -0,0 +1,138 @@
+
+With Azure App Service, developers can deploy applications in a variety of programming languages and application architectures. For YB Shoes, we will create a web app using Node.js.
+
+From the Azure portal, create a web app with Node.js as the runtime stack. This allows Azure to configure the runtime environment with Node.js and the dependencies required to run our application.
+
+![Create a web app with the Node.js runtime in Azure App Service.](/images/tutorials/azure/azure-app-service/azure-web-app.png "Create a web app with the Node.js runtime in Azure App Service.")
+
+As we're deploying our application to Azure from a local repository on our machine, we can forego all settings related to continuous deployment and GitHub Actions. However, for production applications, these integrations greatly improve the development process by deploying applications to Azure automatically when the repository is updated, without the need for manual intervention.
+
+Additionally, by selecting a free or basic tier of Azure App Service, we will deploy our application to a single zone by default. Premium pricing plans allow for multi-zone deployments for increased availability.
+
+After creating your application, visit the associated URL to confirm it's up and running!
+
+![Application running in Azure and waiting for content.](/images/tutorials/azure/azure-app-service/azure-app-running.png "Application running in Azure and waiting for content.")
+
+Now that we've set the foundation on Azure, let's deploy a database cluster for YB Shoes on YugabyteDB Managed.
+
+## Get Started on YugabyteDB Managed
+
+Visit our documentation site to create a YugabyteDB cluster on Azure.
+
+I've chosen to deploy a three-node cluster to Azure, in the WestUS3 region. This configuration provides fault tolerance across availability zones. However, you can start with an always-free single-node cluster.
+
+![Create free YugabyteDB single-region cluster.](/images/tutorials/azure/azure-app-service/yb-create-cluster.png "Create free YugabyteDB single-region cluster.")
+
+Add the [outbound addresses for your web app](https://learn.microsoft.com/en-us/azure/app-service/overview-inbound-outbound-ips), which can be found in the networking tab of the Azure portal. This will ensure that a connection can be made between Azure App Service and YugabyteDB.
+
+![Locate the outbound IP addresses for your web app in Azure.](/images/tutorials/azure/azure-app-service/azure-outbound-traffic.png "Locate the outbound IP addresses for your web app in Azure.")
+
+Now that we have a working cluster in YugabyteDB Managed, let's add some data.
+
+## Add data to YugabyteDB
+
+Now that our cluster is running in the cloud, we can seed it with data using the provided `schema.sql` and `data.sql` files.
+
+1. Use the [YugabyteDB Cloud Shell](https://docs.yugabyte.com/preview/yugabyte-cloud/cloud-connect/connect-cloud-shell/)to connect to your cluster.
+1. Execute the `schema.sql` script against your cluster.
```

Comment:
What are the actual commands? \i meta-command? 
Also, does this actually work with cloud shell, there were restrictions on accessing file system, and how do they access the scripts?